xqueue_watcher/client.py

```diff
@@ -158,12 +166,13 @@ def run(self):
         """
         Run forever, processing items from the queue
         """
+        login_sleep_duration = self.wait_duration * 5
```

Comment:
Where did the 5 come from? Perhaps move it to a named constant to avoid 'magic numbers'.

Reply:
Previously magic number that I set to be a calculated rather than hardcoded magic number 😄  I'll add a config value